rest_framework_tracking/base_mixins.py

```diff
@@ -91,6 +101,15 @@ def finalize_response(self, request, response, *args, **kwargs):
             )
             if self._clean_data(request.query_params.dict()) == {}:
                 self.log.update({"query_params": self.log["data"]})
+            
```

Comment:
maybe more efficient to check if geolocation required further up, say before the previous self.log.update() and include the city/country in the same self.log.update() call

Reply:
fixed it in this commit https://github.com/lingster/drf-api-tracking/pull/38/commits/7011e57b4df29b10f07c101d8341bbff6d612567